docs/src/main/sphinx/object-storage/legacy-azure.md

```diff
@@ -192,3 +192,59 @@ CREATE TABLE hive.abfs_export.orders_abfs
 WITH (format = 'ORC')
 AS SELECT * FROM tpch.sf1.orders;
 ```
+
+(legacy-azure-migration)=
+## Migration to native Azure implementation
+
+Trino includes a {doc}`native Azure implementation
+</object-storage/file-system-azure>` that does not rely on the legacy Hadoop
+implementation. Trino recommends upgrading existing deployments to the new
+native implementation at your earliest convenience.
+
+The following considerations apply to legacy configurations moving to the
+native implementation:
+
+* Legacy storage types (WASB and ADL) are not supported.
+* Azure Government is not supported.
```

Comment:
Thats no longer true with the new PR from David https://github.com/trinodb/trino/pull/23071 which will probably merge before this PR